styles/components/registerV3.module.css

```diff
@@ -247,73 +241,81 @@
   opacity: 0.7;
 }
 
-.stepsContainer {
-  min-width: 220px;
-  display: flex;
-  position: relative;
-  flex-direction: column;
-  justify-content: space-between;
-  border: 1px solid #4545451A;
-  border-radius: 16px;
-  box-shadow: 0px 2px 30px 0px rgba(0, 0, 0, 0.06);
-  background-color: #FFFFFF
+.purchaseStepNav {
+  display: none; 
 }
 
-.stepsBackground {
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  height: 15%;
+.purchaseStepNavMobile {
+  display: block;
+  background-color: white;
+  border: 1px solid #4545451A;
   border-radius: 16px;
   overflow: hidden;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-.progressSteps {
+@media (min-width: 768px) {
+  .purchaseStepNav {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 200px; 
+    flex-shrink: 0; 
+    background-color: white;
+    border: 1px solid #4545451A;
+    border-radius: 16px;
+    overflow: hidden;
+  }
+
+  .purchaseStepNavMobile {
+    display: none; 
+  }
+}
+
+.stepsContainer {
+  padding: 12px;
   display: flex;
   flex-direction: column;
-  align-items: center;
-  text-align: center;
-  padding: 24px;
-  gap: 24px;
+  gap: 10px; 
+  max-width: 100%;
+  min-width: 220px;
 }
 
-.progressStep {
-  width: 100%;
+.step {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
   align-items: center;
-  text-align: center;
-  gap: 5px;
   cursor: pointer;
-  color: var(--secondary);
+  padding: 12px;
+  justify-content: space-between;
+  margin-right: 12px;
 }
 
-.progressStepName {
+.stepContent {
   display: flex;
   align-items: center;
   gap: 8px;
-  font-size: 14px;
-  font-weight: 700;
+}
+
+.passedStep {
+  color: var(--primary, #19aa6e); 
```

Comment:
I know figma always generate the variables like that, but please don't specify the color, only keep the variable name: `var(--primary, #19aa6e)`. The aim of using variables is that we can easy change the whole website's style.

You'll need to check the variables are well defined inside styles/globals.css (in :root). Please apply this change to the others variable you use